aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -12,6 +12,24 @@
 namespace at {
 namespace native {
 
+void fill_depthwise_conv_desc(MPSGraphDepthwiseConvolution3DOpDescriptor* descriptor_,
+                    NSUInteger strideInX, NSUInteger strideInY,
+                    NSUInteger dilationRateInX, NSUInteger dilationRateInY,
+                    NSUInteger paddingHorizontal, NSUInteger paddingVertical,
+                    c10::MemoryFormat memory_format, NSUInteger groups) {
+  descriptor_.strides = @[@1, [[NSNumber alloc] initWithInteger: strideInY],
+                              [[NSNumber alloc] initWithInteger: strideInX]];
+  descriptor_.dilationRates = @[@1, [[NSNumber alloc] initWithInteger: dilationRateInY],
```

Comment:
For better readability, can we replace all of these allocations of `[NSNumber alloc] initWithInteger: X` with just `@X`? Like this one:
`descriptor_.strides = @[@1, @(strideInY), @(strideInX)];`

Reply:
This is for existing test cases.. we remap the existing Conv to depth wise for improved performance. It already has bunch of conv tests.

Reply:
The cleanup I can do while upstreaming..